docs/quick_start/qourier/index.md

```diff
@@ -3,4 +3,4 @@
     on [GitHub](https://github.com/QuEraComputing/bloqade/issues/new) if you need help or want to
     contribute.
 
-Hardware level moves. Included as an example "extra" dialect set. !! To be removed before open open source !!
\ No newline at end of file
+Hardware level moves. Included as an example "extra" dialect set. !! To be removed before open open source !!
```

Comment:
I assume we can remove this note now?

Reply:
My linter decided to update this file I think we just leave it as is for now.